contrib/platform/src/com/sun/jna/platform/win32/Wtsapi32.java

```diff
@@ -86,6 +95,61 @@
 	 */
 	int WTS_SESSION_REMOTE_CONTROL = 0x9;
 
+    /**
+     * Specifies the current server
+     */
+    HANDLE WTS_CURRENT_SERVER_HANDLE = new HANDLE(null);
+
+    /**
+     * Specifies the current session (SessionId)
+     */
+    DWORD WTS_CURRENT_SESSION = new DWORD(-1L);
+
+    /**
+     * Specifies any-session (SessionId)
+     */
+    DWORD WTS_ANY_SESSION = new DWORD(-2L);
+
+    int WTS_PROCESS_INFO_LEVEL_0 = 0;
+    int WTS_PROCESS_INFO_LEVEL_1 = 1;
+
+    /**
+     * Contains extended information about a process running on a Remote Desktop
+     * Session Host (RD Session Host) server. This structure is returned by the
+     * WTSEnumerateProcessesEx function when you set the pLevel parameter to
+     * one.
+     * 
+     * @see <A HREF=
+     *      "https://docs.microsoft.com/en-us/windows/desktop/api/wtsapi32/ns-wtsapi32-_wts_process_info_exa">WTS_PROCESS_INFO_EXA</A>
+     * @see <A HREF=
+     *      "https://docs.microsoft.com/en-us/windows/desktop/api/wtsapi32/ns-wtsapi32-_wts_process_info_exw">WTS_PROCESS_INFO_EXW</A>
+     */
+    @FieldOrder({ "SessionId", "ProcessId", "pProcessName", "pUserSid", "NumberOfThreads", "HandleCount",
+            "PagefileUsage", "PeakPagefileUsage", "WorkingSetSize", "PeakWorkingSetSize", "UserTime", "KernelTime" })
+    class WTS_PROCESS_INFO_EX extends Structure {
+        public DWORD SessionId;
```

Comment:
Any reason not to define this as `int`? Same goes for other DWORDs introduced here.

Reply:
Same comment I just noted on another PR. I went back and forth a few times between `int` and `DWORD`.  What sold me on the latter was reading a message thread about Windows Process IDs, and seeing a comment from a Microsoft Engineer that they actually had seen ProcessID values above `Integer.MAX_VALUE`.     Is there a general policy we follow for JNA mappings?  I understand primitives would be more efficient.

Reply:
Your argument is sound, see the other PR for my comment on it.

Reply:
I'm ok leaving it as is unless/until the JNA project decides to convert ALL the code.  As-is, it's somewhat of a mixed bag (I've seen both styles in existing code) and users must generally be responsible.   Might be a good subject to put in the Contributing document.